components/HeaderDesktop.tsx

```diff
@@ -232,12 +284,13 @@ function Language({ href, children }: LanguageProps) {
   return (
     <Link href={href}>
       <a
-        className="flex items-center justify-between rounded-xl hover:bg-gray-dark p-3 pt-4 font-semibold"
+        className="grid auto-rows-fr items-center text-left rounded-xl hover:bg-gray-dark p-3 pt-4 font-semibold w-full"
```

Comment:
`text-left` is the default behavior. Perhaps we should get rid of `text-center` at line 30 instead of using `text-left` here.

Reply:
Same with other occurrences of `text-left`.

Reply:
If i delete `text-left` the text is no more at left

Reply:
You need to remove `text-center` at line 30

Reply:
Then, you can remove all instances of `text-left`, I believe.